packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,129 @@
+  - [Setting up the RNTester E2E folder](#setting-up-the-rntester-e2e-folder)
+- [Testing the RNTester app E2E](#testing-the-rntester-app-e2e)
+- [Adding new tests (and project structure)](#adding-new-tests-and-project-structure)
+
+## Setting up
+
+### (one-off) Setting up Appium
+
+The first step you need to do is to ensure to install the tooling:
+
+```bash
+npm install appium@2.0.0-beta.71 -g
+appium driver install uiautomator2
+appium driver install xcuitest
+```
+
+> More details about drivers in Appium [here](https://appium.github.io/appium/docs/en/2.0/guides/managing-exts/) and [here](https://appium.github.io/appium/docs/en/2.0/quickstart/uiauto2-driver/)
+
+You should not need to run install commands for drivers separately more than once, even if you bump the dep in package.json.
+
+### Building RNTester app
+
+Building manually *.app* and *.apk* is required to run automation tests on local environment.
+
+0. *(optional)* If you previously built RNTester, you may need to clean up build files and Pods:
+
+    ```bash
+    yarn test-e2e-local-clean
+    ```
+
+1. Step 1: install node modules for the repository, then navigate in the rn-tester folder
+
+    ```bash
+    cd react-native
+    yarn install
+    cd packages/rn-tester
+    ```
+
+Now, depending on the platform, there are some specific steps
+
+#### Building for iOS
+
+0. Make sure you have Bundler `gem install bundler` - we use it ensure installing the right version of CocoaPods locally.
+1. Install Bundler and CocoaPods dependencies: `bundle install` then `bundle exec pod install` or `yarn setup-ios-hermes` for RNTester with Hermes. In order to use JSC instead of Hermes engine, run: `USE_HERMES=0 bundle exec pod install` or `setup-ios-jsc` instead.
+2. Open the generated `RNTesterPods.xcworkspace`.
+3. Build the app via XCode.
+4. Find the **RNTester.app** in `~/Library/Developer/Xcode/DerivedData/RNTesterPods-{id}/Build/Products/Debug-iphonesimulator`
+5. Copy the app to the following directory `/react-native/packages/rn-tester-e2e/apps`.
+6. change its name name to: `rn-tester.app`
+
+#### Building for Android
+
+0. You'll need to have all the [prerequisites](https://reactnative.dev/contributing/how-to-build-from-source#prerequisites) (SDK, NDK) for Building React Native installed.
+1. Start an Android emulator.
+2. Build the app via
+
+    ```bash
+    # In order to not use Hermes engine, run `yarn install-android-jsc` instead.
+    yarn install-android-hermes
+    yarn start
+    ```
+
+    *Note: Building for the first time can take a while.*
+
+3. Find the **RNTester.app** in `~/react-native/packages/rn-tester/android/app/build/outputs/apk/hermes/debug`
+4. copy the app `app-*-debug.apk` to the following directory `/react-native/packages/rn-tester-e2e/apps`
+5. change its name name to: `rn-tester.apk`
+
+### Setting up the RNTester E2E folder
+
+In `react-native/packages/rn-tester-e2e` install the needed dependencies via:
+
+```bash
+yarn install
```

Comment:
Out of ignorance, but if the e2e folder is in a workspace, shouldn't `yarn` run in the `react-native` root folder also install all the dependencies in the `rn-tester-e2e` folder? 🤔